vcpkg/consume/binary-caching-default.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Default local vcpkg binary cache"
+description: Learn about the default vcpkg binary cache
+author: vicroms
+ms.author: viromer
+ms.date: 9/25/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+# Default local vcpkg binary cache
+
+By default, vcpkg creates a local binary cache. This saves time when installing a package that has been previously installed in the same machine.
+
+The location of the default binary cache depends on your operating system.
+
+## [Windows](#tab/default-windows)
+
+By default, binary caching is enabled at the first valid location among the following:
+
+* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
+* `%LOCALAPPDATA%\vcpkg\archives`
+* `%APPDATA%\vcpkg\archives`
+
+## [Unix](#tab/unix)
+
+By default, binary caching is enabled at the first valid location among the following:
+
+* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
+* `$XDG_CACHE_HOME/vcpkg/archives`
+* `$HOME/.cache/vcpkg/archives`
+
```

Comment:
```suggestion
---
title: "Default local vcpkg binary cache"
description: Learn about the default vcpkg binary cache
zone_pivot_group_filename: zone-pivot-groups.json
zone_pivot_groups: operating-systems
author: vicroms
ms.author: viromer
ms.date: 9/25/2023
ms.prod: vcpkg
ms.topic: concept-article
---
# Default local vcpkg binary cache

By default, vcpkg creates a local binary cache. This saves time when installing a package that has been previously installed in the same machine.

::: zone pivot="os-windows"

By default, binary caching is enabled at the first valid location among the following:

* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
* `%LOCALAPPDATA%\vcpkg\archives`
* `%APPDATA%\vcpkg\archives`

::: zone-end
::: zone pivot="os-linux, os-macos"

By default, binary caching is enabled at the first valid location among the following:

* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
* `$XDG_CACHE_HOME/vcpkg/archives`
* `$HOME/.cache/vcpkg/archives`
:::zone-end
```

Reply:
If you choose to use the article pivots, simply add the `zone_pivot_group_filename: zone-pivot-groups.json`
`zone_pivot_groups: operating-systems` to the attributes, and wrap the windows related code in `::: zone pivot="os-windows" .... ::: zone-end`. Similarily for `os-linux` and `os-macos`.